src/main/java/org/folio/innreach/domain/entity/InnReachTransaction.java

```diff
@@ -72,6 +84,17 @@
     "JOIN FETCH h.pickupLocation " +
     "WHERE irt.trackingId = :trackingId AND irt.centralServerCode = :centralServerCode";
 
+  private static final String BY_SHIPPED_ITEM_BARCODE_AND_STATE_IN_QUERY_POSTFIX = "WHERE th.shippedItemBarcode = :shippedItemBarcode AND irt.state IN (:states)";
```

Comment:
Is it going to work with ITEM_HOLD transactions? There is no **shippedItemBarcode** field in `TransactionItemHold` model

Reply:
@oliinyko nope) I've updated the JPQL to get 'PATRON' transactions only

Reply:
@andrew-khodyka but it looks like this type of transaction should be supported somehow) 

> Item Hold (checkout to borrowing site): ITEM_HOLD or TRANSFER